examples/misc/test/language_tour/collections/null_aware_element_b.dart

```diff
@@ -0,0 +1,34 @@
+// ignore_for_file: avoid_init_to_null, invalid_null_aware_operator
+import 'package:test/test.dart';
+
+void main() {
+  // #docregion code_sample
+  String? itemX = 'Apple';
+  String? itemY = null;
+  
+  int? quantityX = 3;
+  int? quantityY = null;
```

Comment:
I wonder if this would be easier to read with different names, like:

```dart
String? presentKey = 'Apple';
String? absentKey = null;

int? presentValue = 3;
int? absentValue = null;
```

Reply:
Updated!